src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,472 @@
+
+package com.lftechnology.java.training.krishna.employeemanagement.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import com.lftechnology.java.training.krishna.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.krishna.employeemanagement.domain.User;
+import com.lftechnology.java.training.krishna.employeemanagement.service.employee.impl.EmployeeServiceImpl;
+import com.lftechnology.java.training.krishna.employeemanagement.service.user.impl.UserServiceImpl;
+import com.lftechnology.java.training.krishna.employeemanagement.utils.Role;
+import com.lftechnology.java.training.krishna.employeemanagement.utils.ValidationUtils;
+
+/**
+ *  The application should have the following features:
+ * 1. Application will have two users: a. admin user b. normal user
+ * 2. Admin user will have option to add/delete user
+ * 3. Normal user can view/search employee
+ * 4. admin user will be generated by the system by default
+ * 5. admin user will add normal user
+ */
+
+/**
+ * This program used for simple employeement system
+ *
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class LoginController {
+
+	public static UserServiceImpl userServiceImpl;
+	public static EmployeeServiceImpl employeeServiceImpl;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make employeeServiceImpl a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static employeeServiceImpl" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)